appveyor.yml

```diff
@@ -56,3 +55,35 @@ build_script:
     - copy paths.props.appveyor paths.props
     - msbuild PluginManager.sln /m /p:configuration="%configuration%" /p:platform="%platform%" /p:PlatformToolset="%PlatformToolset%"
 
+after_build:
+    - cd "%APPVEYOR_BUILD_FOLDER%"
+    - ps: >-
+        if ($($env:APPVEYOR_REPO_TAG) -eq "true" -and $env:CONFIGURATION -eq "Release" -and $env:PLATFORMTOOLSET -eq "v120_xp") {
+            if ($env:PLATFORM -eq "x64" -and $env:CONFIGURATION -eq "Release") {
+                $PluginManagerZipFileName = "PluginManager_$($env:APPVEYOR_REPO_TAG_NAME)_x64.zip"
+                7z a $PluginManagerZipFileName .\NotepadTest\x64\*
+            }
+
+            if ($env:PLATFORM -eq "Win32" -and $env:CONFIGURATION -eq "Release") {
+                $PluginManagerZipFileName = "PluginManager_$($env:APPVEYOR_REPO_TAG_NAME)_UNI.zip"
+                7z a $PluginManagerZipFileName .\NotepadTest\unicode\*
+            }
+        }
+
+artifacts:
+  - path: PluginManager_*.zip
+    name: releases
+
+deploy:
+    provider: GitHub
+    auth_token:
+        secure: veUALhgjhvRE0vKfocbs5hJWTc6sZuWf8GHp/LOTVr8SKi94WGITV21x6JilKwie
```

Comment:
this maybe needs to be updated, currently a encrypted token from me, see
https://www.appveyor.com/docs/deployment/github/#provider-settings